pkg/cmd/codespace/list.go

```diff
@@ -88,8 +102,25 @@ func (a *App) List(ctx context.Context, limit int, exporter cmdutil.Exporter) er
 		} else {
 			tp.AddField(c.CreatedAt, nil, nil)
 		}
+
+		if hasNonProdVSCSTarget {
+			tp.AddField(c.VSCSTarget, nil, nil)
+		}
+
 		tp.EndRow()
 	}
 
 	return tp.Render()
 }
+
+func formatNameForVSCSTarget(name, vscsTarget string) string {
+	if vscsTarget == "dev" || vscsTarget == "local" {
```

Comment:
I think the API response for dev is `development` instead of `dev`.

Reply:
Fixed